docker-plugin/src/main/resources/com/nirima/jenkins/plugins/docker/DockerJobProperty/config.jelly

```diff
@@ -2,21 +2,22 @@
 <j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:d="jelly:define" xmlns:l="/lib/layout" xmlns:t="/lib/hudson" xmlns:f="/lib/form">
 
     <f:optionalBlock title="Docker Container" name="hasDockerContainer" inline="true" checked="${instance!=null}" >
-        <f:entry title="${%Commit on successful build}" field="tagOnCompletion">
-            <f:checkbox default="false"/>
-        </f:entry>
+        <f:optionalBlock title="${%Commit on successful build}" field="tagOnCompletion" inline="true">
```

Comment:
Optional block in optional block just for some number of values looks like a wrong design